### PR TITLE
Add Jenkinsfile for CI

### PR DESCRIPTION
**Pull request by linhkikuchi**
For https://github.com/brave/devops/issues/8063
Lint is added to CI here https://github.com/brave/devops/pull/8064, so we can remove github action

### CODE REVIEW

**Comment by fmarier** [comment]
It should be fixed by https://github.com/brave/tor_build_scripts/commit/a0f78697a63817b6495bf685e75849568e444fa8

Not sure why that libevent test started failing, but I've disabled it for now.

File: .github/workflows/master.yml
Comment (fmarier) [inline .github/workflows/master.yml:23]: Was this check merged into the Jenkins pipeline?

Comment (linhkikuchi) [reply .github/workflows/master.yml:23]: I added this in Jenkins pipeline in this PR https://github.com/brave/devops/pull/8064/files#diff-016e64047994aa44889af773c7791164370c71bcc0a068c69e22a8f282360096R29, waiting for your approval for this one, so I can merge them together

**Review by fmarier — APPROVED** [review]
Looks good. Please rebase so we can confirm that the build succeeds with the fix I committed today.

**Comment by linhkikuchi** [comment]
CI passed https://ci.brave.com/job/pr-brave-tor-client-build-pr-test-linh-add-jenkinsfile/8//flowGraphTable/?auto_refresh=true

**Pull request merged**
